gramps/gui/plug/_windows.py

```diff
@@ -282,9 +283,7 @@ def __init__(self, dbstate, uistate, track=[]):
         if __debug__:
             # Only show the "Reload" button when in debug mode
             # (without -O on the command line)
-            self.__reload_btn = Gtk.Button(label=_("Reload"))
-            self.window.action_area.add(self.__reload_btn)
-            self.__reload_btn.connect('clicked', self.__reload)
+            self.window.add_button(_("Reload"), Gtk.ResponseType.NONE)
```

Comment:
It may be better to assign a custom response value rather than `Gtk.ResponseType.NONE`.